codec/av1.md

```diff
@@ -28,6 +25,9 @@ The top Matroska element that contains interleaved audio, video, subtitles as we
 ## Temporal Unit
 All the OBUs that are associated with a time instant. It consists of a `Temporal Delimiter OBU`, and all the OBUs that follow, up to but not including the next `Temporal Delimiter OBU`.
 
+## CVS
+A Coded Video Sequence is a sequence of `Temporal Units` where the contents of all `Sequence Header OBUs` in the bitstream before Matroska encapsulation must be bit-identical except for the contents of __[operating_parameters_info]__.
+
 

```

Comment:
Did you just reorder it or change the text ? They are ordered alphabetically. And if that's just reordering then it's not a `Seeking & Sequence Header rules`.

Reply:
I did reorder and change the text. I was not satisfied with the current wording of this. After all, "video frames" could mean: output video frames, (Redundant) Frame (Header) OBUs or Matroska "frames", i.e. (Simple)Blocks. The first of these is not what is meant, the second is also not good because there can be OBUs of a different type in a CVS. And the third possibility simply ignores that CVS are defined in the AV1 specification and that they existed before the AV1 Matroska/Webm mapping. So I thought that I should clarify this. And given that the definition of a CVS is intimately related to what type of AV1 shall be allowed in Matroska I thought that this change belongs right besides the other ones.

As for reordering, I have used `Temporal Units` in the definition of `CVS` so I thought that the latter should be defined after the former.

I think that my second commit should be split from the other ones. Should I force-push an update to my branch (including rewriting history) or do something else?